src/pages/blog.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {Fragment} from 'react';
+import { client } from '../lib/client';
+
+import Header from './components-global/header';
+import Footer from './components-global/footer';
+
+const Blog = ({blogs}) => {
+
+  return (
+    <Fragment>
```

Comment:
@KennedyNvsf please use the shorter syntax for fragments without keys or attributes, <></> , that way you can also remove it from the import